client/components/spinner-button/docs/example.jsx

```diff
@@ -0,0 +1,27 @@
+/**
+ * External dependencies
+ */
+import React, { Component } from 'react';
+
+/**
+ * Internal dependencies
+ */
+import SpinnerButton from '../index';
+
+export default class SpinnerButtonExample extends Component {
```

Comment:
The DevDocs screen is picky about the `displayName` of its components for generating the single component view. For example, this should but does not load at the following URL:

http://calypso.localhost:3000/devdocs/design/spinner-button

The exported component _must_ have a display name of `SpinnerButton` for this to work as expected.